src/Akeneo/Connectivity/Connection/workspaces/permission-form/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/* istanbul ignore file */
+import PermissionFormRegistry, {PermissionFormProvider} from './registry/PermissionFormRegistry';
```

Comment:
```suggestion
/* istanbul ignore file */

import PermissionFormRegistry, {PermissionFormProvider} from './registry/PermissionFormRegistry';
```
You have to leave an empty line after istanbul, it's kinda a bug.